tests/build_disk_index.cpp

```diff
@@ -57,6 +57,9 @@ int main(int argc, char** argv) {
                        po::bool_switch()->default_value(false),
                        "Include full precision data in the index. Use only in "
                        "conjuction with compressed data on SSD.");
+    desc.add_options()(
+        "build_PQ_bytes", po::value<uint32_t>(&build_PQ)->default_value(0),
```

Comment:
@harsha-simhadri , for this parameter should it be the same as the PQ bytes per vector at search time? Like 48d for our 384d experimentation

Reply:
It can be different. I will write in the markdown documentation file